spec/appendix.md

```diff
@@ -1,12 +1,58 @@
 ## Appendix
 
+### Trust Over IP Glossary - Controlled Terms 
+
+NOTE: 
+* The following set of identifiers are from the current [ToIP Glossary](https://docs.google.com/document/d/1fZByfuSOwszDRkE7ARQLeElSYmVznoOyJK4sxRvJpyM/edit?usp=sharing) (Google Doc). These definitions need to be moved over to the (under development) ToIP Terms tool (aka TEv2).
+* TODO: not all ToIP definitions are fully linked here. This whole section is more of an exemplar as of the time of writing.
+
+[[def: autonomous identifier]] 
+~ Another term for [[ref: self-certifying identifier (SCID)]]. 
+
+[[def: cryptographic keys]]
+~ TODO: Replace with ToIP Glossary link
+
+[[def: cryptographically verifiable]]
+~ A property of a data structure that has been digitally signed using a private key such that the digital signature can be verified using the public key. Verifiable data, verifiable messages, verifiable credentials, and verifiable data registries are all cryptographically verifiable. Cryptographic verifiability is a primary goal of the ToIP Technology Stack.
+
+[[def: identifier]]
+~ A single attribute—typically a character string—that uniquely identifies an entity within a specific context (which may be a global context). Examples include the name of a party, the URL of an organization, or a serial number for a man-made thing. Supporting definitions: eSSIF-Lab: a character string that is being used for the identification of some entity (yet may refer to 0, 1, or more entities, depending on the context within which it is being used).
+
+[[def: self-certifying identifier (SCID), SCID, SCIDs]]
+~ A subclass of verifiable identifier ([[ref: VID]]) that is [[def: cryptographically verifiable]] without the need to rely on any [[ref: third party]] for [[ref: verification]] because the [[ref: identifier]] is cryptographically bound to the [[ref: cryptographic keys]] from which it was generated. Also known as: [[ref: autonomous identifier]].
+
+[[def: third party]]
+~ A party that is not directly involved in the trust relationship between a first party and a second party, but provides supporting services to either or both of them.
+
+[[def: verifiable identifier (VID), VID, VIDs]]
+~ An identifier over which the controller can provide cryptographic proof of control.
+See also: decentralized identifier, [ref: self-certifying identifier (SCID)].
+
+[[def: verification]]
+~ An action an agent (of a principal) performs to determine the authenticity of a claim or other digital object using a cryptographic key.
+
+#### Glossary WORK TO BE DONE
```

Comment:
Is the idea to wait for this work to be completed before we merge this? For instance we generally would only merge a PR that adds terminology if the terminology has [[ref:]] updates that use the [[def:]] terminology being added. And I think the TODO section shouldn't be merged but could be converted into a set of issues that we could process over time. We could decide to add some to the Spec 1.0 milestone or not.

Reply:
I would prefer to get the work in now, not wait.

There are enough "things" that need to be cleaned up that we should start immediately

Reply:
From our meeting today:
@darrellodonnell will break these Todos out as issues and we'll decide which belong in 1.0 and 1.1 spec milestones.

Reply:
removed the todo items.